### ISSUE TITLE
Add artists statistics

### ISSUE THREAD

**Issue by matthias4217** [matthias4217/jukebox-ultra-nrv#75]
Most played artist and stuff like this

**Comment by Ardhanaariishvara** [comment]
Very hard with the current implementation that has no uniformisation.

**Comment by matthias4217** [comment]
Indeed, when getting information through youtube-dl, it sometimes get the correct artist, and sometimes it's just the channel which uploaded the music. I was thinking more about "not accurate" statistics. However, as it wouldn't be great, it's definitely not a priority.
